WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderReadingPreferencesDialogFragment.kt

```diff
@@ -40,6 +43,13 @@ class ReaderReadingPreferencesDialogFragment : BottomSheetDialogFragment() {
         return R.style.ReaderReadingPreferencesDialogFragment
     }
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.getSerializableCompat<ReaderReadingPreferencesTracker.Source>(ARG_SOURCE)?.let {
```

Comment:
Hmm, good point, though I think we can expect this Fragment to be called via the `show` and `newInstance` static methods. 

TBH I worry a bit about introducing a forced crash here in onCreate, with something that's not feature-breaking (analytics) in case the system handles the fragment creating/recreation oddly and this crashes on the user's hands because of that.